swap/swap.go

```diff
@@ -382,7 +382,7 @@ func (s *SwapData) ToPrettyPrint() *PrettyPrintSwapData {
 		PeerNodeId:      s.PeerNodeId,
 		Amount:          amount,
 		ShortChannelId:  scid,
-		OpeningTxId:     s.OpeningTxBroadcasted.TxId,
+		OpeningTxId:     s.GetOpeningTxId(),
```

Comment:
Is this something we missed earlier?

Reply:
yes, actually wanted to do that in #44  but appararently I put that here